### PR TITLE
increase custom css priority

### PR DESCRIPTION
**Pull request by ee923925github**

### CODE REVIEW

**Review by hungsu — APPROVED** [review]

File: package.json
Comment (jackkoppa) [inline package.json:3]: Hi @ee923925github - these changes are perfect, but can you remove this version bump here? We'll do a publish after your PR merges

**Comment by jackkoppa** [comment]
Thanks @ee923925github, great stuff!

**Review by jackkoppa — CHANGES_REQUESTED** [review]
Ready to merge once we undo the package.json version update :+1:

Great stuff @ee923925github

**Pull request merged**
